sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -52,6 +61,15 @@ def delete_repository(self, repository, **kwargs):
         :returns: Object containing information about the deleted repository
         :rtype: :class:`~azure.containerregistry.DeletedRepositoryResult`
         :raises: :class:`~azure.core.exceptions.ResourceNotFoundError`
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/sample_create_client.py
```

Comment:
dumb question: does this automatically insert the specified lines of the sample into documentation somewhere?

Reply:
Yes! It will insert the code block into the sphinx documentation under the method/parameters/keywords descriptions.